client/my-sites/themes/theme-showcase.jsx

```diff
@@ -58,6 +58,9 @@ const optionShape = PropTypes.shape( {
 	action: PropTypes.func,
 } );
 
+const findTabFilter = ( tabFilters, filterKey, defaultValue ) =>
```

Comment:
`defaultValue` is always the same (`ALL`). Can we for now put this as member function inside `ThemeShowcase` class (above `getTabFilterFromUrl`)? So that we can just hardcode `this.staticFilters.ALL` there.

Reply:
Done.